resources/type-system/strict-inference.md

```diff
@@ -209,9 +211,10 @@ typedef Callback = void Function(int); // OK
 
 void main() {
   var f = (var a) {};      // Inference failure
+  var g = ([_, __]) {};    // OK
```

Comment:
Why are these parameters optional? If it's not important, then make them non-optional (it's just noise then, and makes people like me start questioning the reason behind it).

Reply:
Just noise, I suppose :). In practice I saw it was often optional parameters defined in type aliases. Fixed.